lib/Model/SerializeModelPass.cpp

```diff
@@ -28,6 +28,8 @@ void writeModel(const model::Binary &Model, llvm::Module &M) {
   NamedMDNode *NamedMD = M.getNamedMetadata(ModelMetadataName);
   revng_check(not NamedMD, "The model has alread been serialized");
 
+  Model.verify(true);
```

Comment:
No need, this is now done in `writeModel`.

Reply:
I'll drop this commit then. But, since rebase would invalidate all the `Done` links in the thread I'll postpone this until we're mostly done with the review.

(I'll keep this open so we don't forget)

Reply:
Done.